README.md

```diff
@@ -70,6 +70,17 @@ Next we'll workshop Tweets and social media images. Feel free to leave this to u
 
 
 ## Appendix
+### Code Review
+Most Dev.to posts involve code in the form of:
+- Embedded Glitch apps
+- Code snippets in Markdown 
+
+We do not have a strict code review process or standards, but we like to see:
+- well-commented code
+- a README.md in all Glitch apps
+- files named in a way that will make sense I.E. "feedparser.js" rather than "file1.js"
+
```

Comment:
Nit: Might be worth linking to the Launch Checklist app, or Lyzi's guidence around Starter apps?